src/components/ListItem.tsx

```diff
@@ -0,0 +1,57 @@
+import styled from '@emotion/styled'
+import { IconType } from 'react-icons'
+
+interface Props {
+  item: string
+  index: number
+  icon: IconType
+}
+
+const Container = styled.div`
+  display: flex;
+  align-items: flex-start;
+  justify-content: space-between;
+  width: 381px;
+  height: 62px;
+  border-bottom: 1px solid #5b5b5b;
+  margin-top: 22px;
+  padding-left: 18px;
+  padding-right: 18px;
+  box-sizing: border-box;
+  flex-shrink: 0;
+`
+
+const Text = styled.p`
```

Comment:
```suggestion
const Text = styled.p`
  position: relative;
  top: 6px;
```

위 코드를 추가로 적용하면 텍스트 상하가 더욱 정렬되어 보여요!
<img width="431" alt="image" src="https://github.com/user-attachments/assets/c6152841-7a18-4752-a6d8-f0eb393e1663">

Reply:
전체 수정 완료되었습니다!